### PR TITLE
Added checksum to Dockerfile

### PR DESCRIPTION
**Pull request by yenicapotediaz**
Fixes: #100
While I was in there I also took care of #143

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -158,10 +164,10 @@ RUN     mkdir -p /etc/helm/plugins/appr && \
         helm registry version placeholder
 
 # Crash application
-RUN     wget -q https://github.com/samsung-cnct/k2-crash-application/releases/download/0.1.0/k2-crash-application_0.1.0_linux_amd64.tar.gz && \
-        tar -zxvf k2-crash-application_0.1.0_linux_amd64.tar.gz  && \
+RUN     wget -q -O crash-app.tgz https://github.com/samsung-cnct/k2-crash-application/releases/download/0.1.0/k2-crash-application_0.1.0_linux_amd64.tar.gz && \
```
Comment (coffeepac) [inline Dockerfile]: can we add a check for the crash-app as well?

File: Dockerfile [hunk 2]
```diff
@@ -66,63 +72,62 @@ RUN     wget -q https://github.com/samsung-cnct/terraform-provider-execute/relea
         mv terraform-provider-execute /usr/bin/
 
 # Adding Terraform CoreOS Box addon
-RUN 	wget -q https://github.com/samsung-cnct/terraform-provider-coreosbox/releases/download/${TF_COREOSBOX_VERSION}/terraform-provider-coreosbox_linux_amd64.tar.gz && \
+RUN 	  wget -q https://github.com/samsung-cnct/terraform-provider-coreosbox/releases/download/${TF_COREOSBOX_VERSION}/terraform-provider-coreosbox_linux_amd64.tar.gz && \
```
Comment (coffeepac) [inline Dockerfile]: extraneous space, knocks the wget to the right of the following commands, would like them to be starting at the same column

File: Dockerfile [hunk 3]
```diff
@@ -66,63 +72,62 @@ RUN     wget -q https://github.com/samsung-cnct/terraform-provider-execute/relea
...
         tar -zxvf terraform-provider-coreosbox_linux_amd64.tar.gz && \
         rm terraform-provider-coreosbox_linux_amd64.tar.gz && \
         mv terraform-provider-coreosbox /usr/bin/
 
 # Adding Terraform Distro Image Selector addon
-RUN 	wget -q https://github.com/samsung-cnct/terraform-provider-distroimage/releases/download/${TF_DISTROIMAGE_VERSION}/terraform-provider-distroimage_linux_amd64.tar.gz && \
+RUN 	  wget -q https://github.com/samsung-cnct/terraform-provider-distroimage/releases/download/${TF_DISTROIMAGE_VERSION}/terraform-provider-distroimage_linux_amd64.tar.gz && \
```
Comment (coffeepac) [inline Dockerfile]: extraneous space

File: Dockerfile [hunk 4]
```diff
@@ -43,18 +49,18 @@ ENV     APP_REGISTRY_PLUGIN_RELEASE=v0.7.0
...
 # Alpine
-
 ADD     build/alpine-builds /alpine-builds
 
 ENV     APK_PACKAGES="bash ca-certificates openssl openssh python py-openssl py-pip py-cryptography py-cffi zip unzip wget util-linux bind-tools"
 ENV     APK_DEV_PACKAGES="gcc g++ git make libffi-dev linux-headers musl-dev libc-dev openssl-dev python-dev unzip mkinitfs kmod mtools squashfs-tools"
 
 RUN     apk add --update --no-cache ${APK_PACKAGES} ${APK_DEV_PACKAGES} && \
-    	/alpine-builds/build-docker.sh && \
+    	  /alpine-builds/build-docker.sh && \
```
Comment (coffeepac) [inline Dockerfile]: not clear from diff if this is vertically aligned, I think its an extraneous space

File: Dockerfile (hunk 1 above)
Comment (yenicapotediaz) [reply Dockerfile]: Done

**Pull request merged**